orders/models.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from warehouses.models import Item
+
+
+class Order(models.Model):
+    """
+    Ordered items made by User.
+    """
+    item_id = models.ForeignKey(Item, on_delete=models.CASCADE)
+    date_ordered = models.DateTimeField(auto_now_add=True)
+    order_number = models.CharField(max_length=20)
+    ordered_by = models.ForeignKey(User)
```

Comment:
Shouldn't this have an `on_delete` value?

Reply:
Addressed here: https://github.com/jino-d/warehouse/pull/1/commits/59ace3fed4a512764e82158d1ee809b3ee45eeb7